frames/src/main/java/com/checkout/frames/view/InputField.kt

```diff
@@ -89,7 +86,7 @@ internal fun InputField(
             enabled = enabled,
             readOnly = readOnly,
             textStyle = mergedTextStyle,
-            cursorBrush = SolidColor(colors.cursorColor(state.isError.value).value),
+            cursorBrush = SolidColor(provideCursorColor(style.colors)),
```

Comment:
How can we get specific error color in case we have state isError = true

Reply:
With new limited access, we are not able to use the internal state to listen when the error state is enabled, so we can create a custom state and do all stuff by ourselves in the future.